lib/Module/Install/Metadata.pm

```diff
@@ -533,6 +534,7 @@ sub __extract_license {
 		'OpenSSL License'                    => 'unrestricted', 1,
 		'SSLeay License'                     => 'unrestricted', 1,
 		'zlib License'                       => 'open_source',  1,
+		'EUPL'                               => 'eupl',         1,
```

Comment:
Module::Install uses the CPAN::Meta spec v1.4.  The list of licenses supported by this spec can be found [here](https://metacpan.org/pod/distribution/CPAN-Meta/lib/CPAN/Meta/History/Meta_1_4.pod#license).